navigation-animation/src/main/java/com/google/accompanist/navigation/animation/AnimatedComposeNavigator.kt

```diff
@@ -69,14 +71,32 @@ public class AnimatedComposeNavigator : Navigator<AnimatedComposeNavigator.Desti
     /**
      * NavDestination specific to [AnimatedComposeNavigator]
      */
+    @Deprecated(
+        message = "Replace with Androidx ComposeNavigator.Destination and change import to" +
```

Comment:
Missing a space at the end of the string

Reply:
Done